docs/bibliography/diffusion-mri.md

```diff
@@ -1,5 +1,34 @@
-# Resources for learning about diffusion MRI
+# Resources for learning the basics of diffusion MRI
+
+## Here as some basic readings to introduuce the field
+
+Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
```

Comment:
Citations that are in the [bibliography.bib file](https://github.com/nrdg/afq-docs/blob/master/bibliography.bib) (in latex .bib format, which you can get from google scholar, by clicking on the "Import into BibTeX" link for any paper it brings up) can be referenced as follows:

```suggestion
Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1) [@wandell2016clarifying]
```